log4j-appender/src/main/java/org/apache/kafka/log4jappender/KafkaLog4jAppender.java

```diff
@@ -63,9 +64,11 @@
     private String saslKerberosServiceName;
     private String clientJaasConfPath;
     private String kerb5ConfPath;
+    private Integer timeout;
 
     private int retries;
     private int requiredNumAcks = Integer.MAX_VALUE;
+    private boolean ignoreExceptions = true;
```

Comment:
`true` as initial value of `ignoreExceptons` changes default behavior. Users might be relying on exceptions being propagated to callers, thus we should not change this.